.gitignore

```
@@ -26,5 +26,5 @@ src/globals/grid/css-gridish/bx-grid.sketch
 # Generated by npm@5 - project currently uses Yarn.
 package-lock.json
 
-
-
+# AAT test results from DAP/Axe
+results/ 
```

This should be `.aat/results`, right, or am I missing something? 🤔

```suggestion
.aat/results
```

---

The `results/` folder is where the tests are going though...

Looking at the config:

```
 // optional - Where the scan results should be saved.
  // Default: results
  outputFolder: '.aat/results',
```

AAT appears to be ignoring the `outputFolder` being set. 🤔 
Just gonna... change the outputFolder to the default to avoid confusion. Weird that it ignores that though.